tests/dat/manifest_validate_multiline_params.yaml

```diff
@@ -32,6 +32,15 @@ packages:
                     param_string_type_and_value_only:
                         type: string
                         value: foo
+                    param_json_type_and_value_only_1:
+                        type: json
+                        value: '{ "name": "Sam", "place": "Shire" }'
+                    param_json_type_and_value_only_2:
+                        type: json
+                        value: { "name": "${USERNAME}", "password": "${PASSWORD}" }
+                    param_json_type_and_value_only_3:
+                        type: json
+                        value: { "name": "$${USERNAME}", "password": "$${PASSWORD}" }
```

Comment:
Thanks for this.  Did we open an spec/docs issue to document this new syntax for string literals (ala Kube, etc.)?

Reply:
yup we have issue #769 and #770 to update spec and doc respectively.